client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx

```diff
@@ -11,12 +12,17 @@ import useStyles from './ContactQuestioningStyles';
 import { OCCUPATION_LABEL, RELEVANT_OCCUPATION_LABEL } from '../PersonalInfoTab/PersonalInfoTab';
 import useContactFields from 'Utils/vendor/useContactFields';
 
+
 const ContactQuestioningCheck: React.FC<Props> = (props: Props): JSX.Element => {
+    const {control , getValues} = useFormContext();
+
     const classes = useStyles();
 
     const { occupations } = useContext(occupationsContext);
-    const { interactedContact, updateInteractedContact } = props;
-    const { isFieldDisabled } = useContactFields(interactedContact.contactStatus);
+    const { index , interactedContact } = props;
+
+    const formValues = getValues().form ? getValues().form[index] : interactedContact
```

Comment:
When there will be a situtation that there wont be a form and will be a prop? If there is, are there diffrences?

Reply:
Same for all other places used

Reply:
the formValues constant is set to be the interactedContact prop (the inital state) when this is the first time loading the element ( i.e. - the form has no values) , afterwards the value is the form's value so that any changes to the form(like changing the status of the investigations to closed will take place immediately